examples/CampaignManagement/ValidateTextAd.cs

```diff
@@ -117,16 +117,17 @@ public void Run(GoogleAdsClient client, long customerId, long adGroupId)
 
                 if (e.Failure != null)
                 {
+                    // Note: Depending on the ad type, you may get back policy violation errors as
+                    // either PolicyFindingError or PolicyViolationError. ExpandedTextAds return
+                    // errors as PolicyFindingError, so only this case is illustrated here. See
+                    // https://developers.google.com/google-ads/api/docs/policy-exemption/overview
+                    // for additional details.
                     e.Failure.Errors
                         .Where(err =>
-                            err.ErrorCode.PolicyFindingError == PolicyFindingError.PolicyFinding ||
-                            err.ErrorCode.PolicyViolationError == PolicyViolationError.PolicyError)
+                            err.ErrorCode.PolicyFindingError == PolicyFindingError.PolicyFinding)
                         .ToList()
                         .ForEach(delegate (GoogleAdsError err)
                         {
-                            // Only one of PolicyFindingDetails or PolicyViolationDetails will be
-                            // populated. PolicyViolationDetails is used by some ad formats, and
-                            // PolicyFindingDetails by others.
                             int count = 1;
                             if (err.Details.PolicyFindingDetails != null)
```

Comment:
Is this condition still necessary?

Reply:
You could get error for other reasons (e.g. field missing, url schema wrong...). That's why this condition check is still needed.

Reply:
Thanks for confirming